class-lifterlms-rest-api.php

```diff
@@ -196,15 +196,16 @@ public function rest_api_controllers_init() {
 	 * @since 1.0.0-beta.8 Load webhooks actions a little bit later: at init 6 instead of init 10,
 	 *                     just after all the db tables are created (init 5),
 	 *                     to avoid PHP warnings on first plugin activation.
+	 * @since [version] Bump minimum required version to 6.0.0-alpha.1.
```

Comment:
```suggestion
	 * @since [version] Bump minimum required version to 6.0.0-alpha.1.
	 *                     Use `llms()` in favor of deprecated `LLMS()`.
```